src/WebJobs.Extensions.DurableTask/DurableOrchestrationClient.cs

```diff
@@ -219,6 +219,27 @@ public override async Task<IList<DurableOrchestrationStatus>> GetStatusAsync(Dat
             return results;
         }
 
+        // TODO implementation to enable paging
+        public override async Task<DurableOrchestrationStatusContext> GetStatusAsync(DateTime createdTimeFrom, DateTime? createdTimeTo, IEnumerable<OrchestrationRuntimeStatus> runtimeStatus, int pageSize, string continuationToken, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            var serviceClient = (AzureStorageOrchestrationService)this.client.ServiceClient;
+            var statusContext = await serviceClient.GetOrchestrationStateAsync(createdTimeFrom, createdTimeTo, runtimeStatus.Select(x => (OrchestrationStatus)x), pageSize, continuationToken, cancellationToken);
```

Comment:
This line is too long. Can we wrap the parameters, one on each line?

Reply:
fixed long line.